students/pinstripezebra/Circle.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sat Mar  3 19:20:59 2018
+
+@author: seelc
+"""
+import math
+
+class Circle(object):
+    radius = 0
+    diameter = 0
+    def __init__(self, radius):
+        self.radius = radius

```

Comment:
I think you would be better off setting defaults in your `init` via `def __init__(self, radius=0):` and the same for you `from_diameter` classmethod.